src/components/SchedulePage.js

```diff
@@ -0,0 +1,78 @@
+import React, { forwardRef } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { useHistory } from 'react-router-dom'
+import MaterialTable from 'material-table'
+import { 
+  AddBox,
+  Clear,
+  Search,
+  FirstPage,
+  LastPage,
+  ChevronRight,
+  ChevronLeft,
+  ArrowDownward,
+  Remove,
+  ViewColumn} from '@material-ui/icons'
+
+const tableIcons = {
+    Add: forwardRef((props, ref) => <AddBox {...props} ref={ref} />),
+    Clear: forwardRef((props, ref) => <Clear {...props} ref={ref} />),
+    FirstPage: forwardRef((props, ref) => <FirstPage {...props} ref={ref} />),
+    LastPage: forwardRef((props, ref) => <LastPage {...props} ref={ref} />),
+    NextPage: forwardRef((props, ref) => <ChevronRight {...props} ref={ref} />),
+    PreviousPage: forwardRef((props, ref) => <ChevronLeft {...props} ref={ref} />),
+    ResetSearch: forwardRef((props, ref) => <Clear {...props} ref={ref} />),
+    Search: forwardRef((props, ref) => <Search {...props} ref={ref} />),
+    SortArrow: forwardRef((props, ref) => <ArrowDownward {...props} ref={ref} />),
+    ThirdStateCheck: forwardRef((props, ref) => <Remove {...props} ref={ref} />),
+    ViewColumn: forwardRef((props, ref) => <ViewColumn {...props} ref={ref} />)
+};
+
+const useStyles = makeStyles({
+  
+});
+
+export default function SchedulePage() {
+  const classes = useStyles();
+  let history = useHistory();
```

Comment:
We probably will use these later on but we can remove them for now to resolve the compiler warnings.